Shared/Settings.swift

```diff
@@ -11,6 +11,7 @@ enum SettingsToggle: String {
     case blockSocial = "BlockSocial"
     case blockOther = "BlockOther"
     case blockFonts = "BlockFonts"
+    case homeScreenTips = "HomeScreenTips"
```

Comment:
Nit but I would change this to be ShowScreenTips so we know if it's true that means we show vs. false we do not show.